docs/src/browsers.md

```diff
@@ -3,101 +3,292 @@ id: browsers
+```
+
+```bash python
+playwright install-deps chromium
+```
+
+```bash csharp
+pwsh bin/Debug/netX/playwright.ps1 install-deps chromium
+```
+
+It's also possible to combine `install-deps` with `install` and install by that the browsers and OS dependencies with a single command. This would do both for Chromium, but you can also leave it out.
+
+```bash js
+npx playwright install --with-deps chromium
+```
+
+```bash java
+mvn exec:java -e -D exec.mainClass=com.microsoft.playwright.CLI -D exec.args="install --with-deps chromium"
+```
+
+```bash python
+playwright install --with-deps chromium
+```
+
+```bash csharp
+pwsh bin/Debug/netX/playwright.ps1 install --with-deps chromium
+```
+## Browser Configuration
+* langs: js
+
+Playwright supports multiple "projects" so that can run your tests in multiple browsers and configurations. Here is an example that runs every test in Chromium, Firefox and WebKit, by creating a project for each. You can also add branded browsers and mobile devices as "projects", see below.
 
 ```js tab=js-js
 // @ts-check
-
-const { defineConfig } = require('@playwright/test');
+const { devices, defineConfig } = require('@playwright/test');
 
 module.exports = defineConfig({
-  use: {
-    channel: 'chrome',
-  },
+  projects: [
+    {
+      name: 'chromium',
+      use: { ...devices['Desktop Chrome'] },
+    },
+    {
+      name: 'firefox',
+      use: { ...devices['Desktop Firefox'] },
+    },
+    {
+      name: 'webkit',
+      use: { ...devices['Desktop Safari'] },
+    },
+  ],
 });
 ```
 
 ```js tab=js-ts
-import { defineConfig } from '@playwright/test';
+import { defineConfig, devices } from '@playwright/test';
+
 export default defineConfig({
-  use: {
-    channel: 'chrome',
-  },
+  projects: [
+    {
+      name: 'chromium',
+      use: { ...devices['Desktop Chrome'] },
+    },
+    {
+      name: 'firefox',
+      use: { ...devices['Desktop Firefox'] },
+    },
+    {
+      name: 'webkit',
+      use: { ...devices['Desktop Safari'] },
+    },
+  ],
 });
 ```
 
-```js tab=js-library
-const { chromium } = require('playwright');
-const browser = await chromium.launch({
-  channel: 'chrome' // or 'msedge', 'chrome-beta', 'msedge-beta', 'msedge-dev', etc.
-});
+You can specify [different options][TestProject] for each project, for example set specific command-line arguments for Chromium.
+
+Playwright Test will run all projects by default.
+
+```bash
+npx playwright test
+
+Running 5 tests using 5 workers
+
+  ✓ [chromium] › example.spec.ts:3:1 › basic test (2s)
+  ✓ [firefox] › example.spec.ts:3:1 › basic test (2s)
+  ✓ [webkit] › example.spec.ts:3:1 › basic test (2s)
+```
+
+Use the `--project` command line option to run a single project.
+
+```bash
+npx playwright test --project=firefox
+
+Running 1 test using 1 worker
+
+  ✓ [firefox] › example.spec.ts:3:1 › basic test (2s)
 ```
 
-```java
-import com.microsoft.playwright.*;
+## Browser versions
```

Comment:
I don't think this header adds much. Perhaps put Chromium et al. as children of "configure browsers"? Maybe we don't need small separate snippets, but instead have the first snippet include projects for all engines, channels and devices?